controllers/upgrade_handlers.go

```diff
@@ -119,6 +119,14 @@ func (u *UpgHandler) PrePivot(ctx context.Context, ibu *lcav1alpha1.ImageBasedUp
 		u.resetProgressMessage(ctx, ibu)
 	}
 
+	u.Log.Info("Running health check for Upgrade (pre-pivot)")
+	if err := CheckHealth(ctx, u.NoncachedClient, u.Log); err != nil {
+		msg := "Waiting for system to stabilize before Upgrade (pre-pivot) stage can continue"
+		u.Log.Error(err, msg)
```

Comment:
Perhaps u.Log.Info instead? Since we are just waiting for the stability, logging as Error may cause confusion and introduce too much noise?